app/controllers/people_controller.rb

```diff
@@ -8,19 +7,12 @@ def index
 
   # GET /people/1
   def show
-    #@current_measurement = @current_user.current_measurement
-    #@latest_bmi = @current_measurement.bmi
-
-    #@current_user = Person.find(params[:id])
-    #if @current_measurement
-      #@current_user = Person.find_by_name("Mozes Kriebel")
-     # @current_measurement = current_user.measurements.first
-     # @latest_bmi = Measurement.bmi_calc(@current_measurement) #if @current_user.measurements.last
   end
 
   # GET /people/new
   def new
     @person = Person.new
+    #NOTE Maybe I need to do it like this when I implement search feature
```

Comment:
This is a good point. You might also want to have a look at [RailsCasts #393](http://railscasts.com/episodes/393-guest-user-record) ;-)